backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/exportimport/ImportVmFromExternalProviderCommand.java

```diff
@@ -75,6 +75,7 @@
 
     private static final Pattern VMWARE_DISK_NAME_PATTERN = Pattern.compile("\\[.*?\\] .*/(.*).vmdk");
     private static final Pattern DISK_NAME_PATTERN = Pattern.compile(".*/([^.]+).*");
+    private static final Pattern DISK_ALIAS_ILLEGAL_CHARS_PATTERN = Pattern.compile("[^\\w.-]");
```

Comment:
you allow only word chars? numeric is impossible? can you tell what exactly is illegal?

Reply:
`\w` includes latin letters, digits and underscore. Also period (`.`) and hyphen (`-`) are allowed. These are characters that we allow for disk alias in WebAdmin.

Reply:
> `\w` includes latin letters, digits and underscore. Also period (`.`) and hyphen (`-`) are allowed. These are characters that we allow for disk alias in WebAdmin.

OK

Reply:
> `\w` includes latin letters, digits and underscore. Also period (`.`) and hyphen (`-`) are allowed. These are characters that we allow for disk alias in WebAdmin.

so can we reuse the pattern that is used by the webadmin?